apps/server/prisma/seed.ts

```diff
@@ -0,0 +1,188 @@
+import { PrismaClient } from '@prisma/client';
+import { v4 as uuidv4 } from 'uuid';
+
+const prisma = new PrismaClient();
+
+const LEADERSHIP_TEAM_UUID = uuidv4();
+const CHIEF_OF_STAFF_UUID = uuidv4();
+const CHIEF_FIN_OFFICER_UUID = uuidv4();
+const AGG_DIR_UUID = uuidv4();
+const CHIEF_LEARNING_ENGAGEMENT_UUID = uuidv4();
+
+async function main() {
+
+  const formTemplate1Id = uuidv4();
+  const formTemplate1 = await prisma.formTemplate.upsert({
+    where: { id: formTemplate1Id },
+    update: {},
+    create: {
+      id: formTemplate1Id,
+      name: 'Form Template 1',
+      formDocLink: 'https://www.mfa.org/'
+    },
+  });
+
+  const signatureFields = await fetchSignatureFields(formTemplate1Id);
+
+  const departmentLeadershipTeam = await prisma.department.upsert({
+    where: { id: LEADERSHIP_TEAM_UUID },
+    update: {},
+    create: {
+      id: LEADERSHIP_TEAM_UUID,
+      name: 'Leadership Team',
+    },
+  });
+  const positions = [
+    {
+      id: CHIEF_OF_STAFF_UUID,
+      name: 'Chief of Staff',
+      departmentId: departmentLeadershipTeam.id,
+      signatureFields: signatureFields,
```

Comment:
right now after running the seed command this line outputs `signatureFields: [ [Object], [Object], [Object] ]` , not sure if that matters or if we'd want to see what's being put into that field

Reply:
https://stackoverflow.com/questions/10729276/how-can-i-get-the-full-object-in-node-jss-console-log-rather-than-object

there may be some missing fields, so we might need to specify what you want to include in the signature fields as well, since we can nest the `include` logic?

Reply:
resolved by using JSON.stringify(allPositions) and seems to be working now